integration_tests/broker_test.go

```diff
@@ -114,10 +188,30 @@ var _ = Describe("Broker", func() {
 			path,
 			bytes.NewBuffer(bindingDetailsJson),
 		)
-		resp := serveRequest(responseRecorder, req)
-		Expect(responseRecorder.Code).To(Equal(http.StatusInternalServerError))
-		body := readResponseBody(resp)
-		Expect(string(body)).To(ContainSubstring("Can't bind an instance"))
+		brokerAPI.ServeHTTP(responseRecorder, req)
+		Expect(responseRecorder.Code).To(Equal(http.StatusCreated))
+
+		var data struct {
```

Comment:
> We deliberately didn't do that because it would introduce circularity, and mean that we wouldn't be effectively testing the JSON serialisation.

I'm not sure I follow. Could you elaborate?